### DISCUSSION TITLE
ReGaHss ERROR on CCU3 - ist Home Assistant die Ursache?

### DISCUSSION THREAD

**Discussion by schnu240** [SukramJ/aiohomematic#1455] [Q&A]
Hi,
seit ein paar Tagen habe ich das Problem das meine CCU3 sich ständig aufhängt. Mal nach 24h mal nach 1-2 Tagen. Meine Integration mit Home Assistant funktioniert seit ein paar Wochen problemlos. Der einzige Unterschied ist das bei mir natürlich immer wieder mal eine neue Automation hinzukommt. 
Im Log der CCU3 sehe ich das es wohl massiv zu **_ReGaHss: ERROR: XMLRPC 'getValue' call failed_** kommt. Diese Fehler scheinen die CCU so lange zu überfluten (mehrere Hundert, oder Tausend) bis sie irgendwann sich einfach aufhängt. Jetzt frage ich mich ob es mit der Homematic Integration zu tun hat. Laut Doku scheint es aber so zu sein das die Home Assistant Variante garkein ReGaHss Interface nutzt? 

Bevor ich zum Home Assistant gewechselt bin hatte ich eine Testinstallation mit iOBroker. Dort hat aber der ReGaHss dafür gesorgt das ich nie eine laufende Installation hinbekommen habe. Dort konnte ich auf jeden Fall feststellen das die iOBroker Integration meine CCU3 abschießt.

Da ich Home Assistant aber zu schätzen gelernt habe möchte ich natürlich nicht mehr wechseln. Ich hoffe ihr könnt mir mit eurem Verständnis über diese Integration helfen den Fehler einzugrenzen.

Gerne liefere ich alle fehlenden Informationen.

**Comment by SukramJ** [comment]
Ich denke es macht mehr Sinn, wenn du dich mit deinem Problem an ein CCU [Expertenforum](https://homematic-forum.de/) wendest. Falls du RaspberryMatic verwendest bist Du natürlich auch in deren [GitHub](https://github.com/jens-maus/RaspberryMatic) sehr gut aufgehoben.

**Comment by schnu240** [comment]
Ja dort werde ich auch suchen aber ich würde gerne mit den Experten hier grundsätzlich gerne einmal verstehen ob ich mit meinem Automatisierungen für Fehler sorgen kann. Werden die Zustände nur zyklisch von der CCU geholt und in den Identitäten gespeichert?
D.h. eine Abfrage in Automatisierungen eines Zustandes erzeugt keinen zusätlichen Request von der CCU?

**Reply by SukramJ** [reply]
Die Zustände müssen nur nach dem Start geladen werden, wobei nur für die wenigsten Entitäten ein getValue benötigt wird.
Was Du in deinen Automatisierungen machst kann ich dir natürlich nicht beantworten, aber das Verwenden eines Zustandes löst keine CCU-Abfrage aus.
Mir ist auch nicht bekannt, das die Integration Probleme auf der CCU verursacht, ausser die CCU hat schon eigene Probleme, wie z.Bsp. eine defekte SD-Karte.
